resources/external.sh

```diff
@@ -18,13 +18,16 @@ retrieve_external_model() {
 
   local source=`urldecode $(<external_source)`
 
-  if [[ -z "$FLAG_CLEAN" ]]; then
-    verify_external_target_folder_is_clean
```

Comment:
I assume that this line is what was referred to by this?

> Note: also tweaked the external files build to avoid rebuilds failing.

Reply:
Yes